content/docs/user-guide/packaging/project-export/project-export-cli.md

```diff
@@ -17,6 +17,46 @@ To make best use of the `export-project` command, it is recommended to have a pr
 To learn more about the AssetBundler and Seed Files, please visit the [overview page on the AssetBundler tool](https://docs.o3de.org/docs/user-guide/packaging/asset-bundler/overview/).
 
 ## Getting Started
+
+### Quick Project Export
+For any project that you want to quickly export, you can do this by running the `export.bat` script (or running the `export.sh` script on Linux) in the root of your project folder.
+
+New projects created from either the `DefaultProject` or `MinimalProject` templates will contain project export helper scripts that simplifies the process to export these projects. These scripts provide a simple base-line for the arguments and settings that are usually needed to export projects when using the O3DE `export-project` command line tool.
+
+If you need to run it from the command line, simply run the following command (assuming project has already been created and registered):
```

Comment:
How else do you run it? Also, project will be created (it exists on your hard drive). It will be always be registered if you created it. Registration is only an issue if you downloaded it from someone else. Creating a project via project manager or o3de.py create-project will register it for you.
```suggestion
Run it from the command line:
```

Reply:
Looks like the original statement about double clicking the batch file was removed, so that line no longer makes sense. I'll take your suggestion.